packages/dds/sequence/src/test/snapshotVersion.spec.ts

```diff
@@ -124,6 +124,17 @@ describe("SharedString Snapshot Version", () => {
     }
     generateSnapshotRebuildTests();
 
+    function generateOldSnapshotRebuildTests() {
+        describe("Snapshot rebuild for old format", () => {
+            for (const str of generateTestStrings()) {
+                if (str[2]) {
+                    generateSnapshotRebuildTest(str[0], str[1]);
+                }
+            }
+        });
+    }
+    generateOldSnapshotRebuildTests();
```

Comment:
I think we want to generate snapshot rebuild tests for all of these cases, no? It's just the snapshot diff test that will fail (since loading and reserializing the snapshot using a standard SharedString won't work)

Reply:
if we don't want to generate all those tests, then we shouldn't be writing the files either. It would probably make more sense to add it as another yield result from existing generateStrings() if we don't want all those tests. I think for the current needs that might be simpler, since interval collection write format doesn't really interact much with sharedstring merge-tree write format (so testing the combinatorial product of the two versions isn't much more interesting than the sum)